ska_conda/pkg_defs/ska3-template/build.sh

```diff
@@ -4,5 +4,10 @@ for file in flt_envs ska_envs.sh ska_envs.csh
    do
     chmod +x ${RECIPE_DIR}/bin/${file}
     cp -a ${RECIPE_DIR}/bin/${file} ${PREFIX}/bin
-    sed -i ""  "s|%{PREFIX}%|${PREFIX}|" ${PREFIX}/bin/${file}
+    # Replace %{PREFIX}% with the conda build prefix.  It looks like it would
+    # be simpler to just make these three files with the standard conda prefix_placeholder
+    # hard-coded in the template files but this sed operation should be robust if the 
```

Comment:
By prefix_placeholder here, I mean that it does/could work to just hard code "/opt/anaconda1anaconda2anaconda3" as the prefix in the three files, as conda recognizes that prefix placeholder as a thing that should be replaced when actually installing the package.  But doing this does not seem to be advertised in the conda docs and I don't know if prefix_placeholder could change.